lib/cli-engine.js

```diff
@@ -347,7 +346,7 @@ function getCacheFile(cacheFile, cwd) {
  */
 function CLIEngine(options) {
 
-    options = lodash.assign(Object.create(null), defaultOptions, options);
+    options = lodash.assign(Object.create(null), defaultOptions, {cwd: process.cwd()}, options);
```

Comment:
default options already has `cwd`, which is set to `process.cwd()`. why is this change necessary?